src/rhsm/profile.py

```diff
@@ -35,7 +35,7 @@
     yum = None
 
 try:
-    imp.find_module("zypp_plugin")
+    importlib.import_module("zypp_plugin")
```

Comment:
It seems like `imp.find_module()` does not actually import a module, and what a5c2fd9c3b330721297ee8b0b36f7bc004dd3321 changed

Because of this, I think it wouldn't be wise to import `zypp_plugin` again, especially that it isn't actually used (seems like an arbitrary way to say "i'm running on a SUSE system, integrate with zypper"...). It seems a potential equivalent could be `importlib.util.find_spec()`, I'd try it.

Reply:
Oh, right, I've totally missed that, thanks.